specs/light_client/merkle_proofs.md

```diff
@@ -127,8 +154,38 @@ def verify_multi_proof(root, indices, leaves, proof):
     return (indices == []) or (1 in tree and tree[1] == root)
 ```
 
+### MerklePartial
+
+We define:
+
+#### `MerklePartial`
+
+
+```python
+{
+    "root": "bytes32",
+    "indices": ["uint64"],
+    "values": ["bytes32"],
+    "proof": ["bytes32"]
+}
+```
+
+#### `merkle_partial_from_paths`
+
+```python
+def merkle_partial_from_paths(obj, paths):
+    indices = set()
+    for path in paths:
+        indices = indices.union(get_generalized_indices(obj, path))
+    return MerklePartial(
+        root=hash_tree_root(obj),
+        indices=indices,
+        values= mk_multi_proof
```

Comment:
`mk_multi_proof` is undefined, and this function does not set `proof`

Reply:
Removed that section entirely for now. I think the right time to bring it back would be after we have executable SSZ serialization, as then we would be able to actually generate the proofs as part of the spec (right now I'm not sure what the "final" interface in the executable spec would be...)